### ISSUE TITLE
Question: Why doesn't Neon care about the dimensions of an MEI file / image?

### ISSUE THREAD

**Issue by JoyfulGen** [DDMAL/Neon2#1244]
More context:

We've been training our models for MS73 on images reduced to 50% of their size. I've started running e2e OMR workflows of MS73 folios using images that are all reduced by 50%. When I upload the resulting MEI file to Neon, I can pair it with a full-size image, a 50% image, and a 25% image and it all looks exactly the same. But aren't the coordinates in the MEI file absolute? Is this expected behaviour? @yinanazhou we need your help.

Here's the MEI file in question: 
[MEI Encoding - 023.mei.zip](https://github.com/user-attachments/files/17364538/MEI.Encoding.-.023.mei.zip)

Full size, half size, and quarter size images can be found here: https://drive.google.com/drive/folders/1lWcx-iVB74iOaDnCYB9RFvMrrO2aYH1-?usp=sharing

**Comment by yinanazhou** [comment]
This is expected because Neon automatically stretch the background image to fit the size of the MEI file.

**Comment by JoyfulGen** [comment]
@fujinaga I'm taking the liberty of tagging you, because you were particularly intrigued by this! Can I close the issue?

**Comment by fujinaga** [comment]
Yes, you can close this issue, but the MEI files that were reduced should be expanded to reflect the original size, by the end of the e2e workflow. This could be done, even after you correct errors in Neon, I guess. You need to do this because, not all editors may be as smart as Neon!  :-)

**Comment by JoyfulGen** [comment]
Duly noted! @kyrieb-ekat this is for you too. We can just use the `MEI resizing` job!

**Comment by kyrieb-ekat** [comment]
Oh that's spectacular news! 

I'll go ahead and close this then, and we can reopen if we want to discuss this further, or we can open a discussion about MEI resizing/workflows etc (e.g., should we just add the ```MEI Resizing Job``` to all OMR workflows to anticipate possible error?, and so on).
